lib/services.sh

```diff
@@ -55,12 +55,10 @@ start_service() {
   for pid in $pids; do kill -9 $pid 2>/dev/null; done
 
   echo "Starting service: $(tput setaf 3)$service$(tput sgr0)"
-
+  export JAVA_OPTS="$java_args \
```

Comment:
Oh man... this is just not very pretty... We are setting the environment variable here but it's the start up script that will use it... I feel that we are asking for bad things to happen one day... (i.e. someone comes along to set JAVA_OPS in the start up script then this max and min mem usage will be overwritten.

Reply:
The `application` plugin (the provider for `distZip`) expects this as an env var - this is just magic you can find in the docs unfortunately. https://docs.gradle.org/current/userguide/application_plugin.html#sec:customizing_start_script_generation